business_requirement_deliverable_sale/models/business.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 Tecnativa Victor M.M. Torres>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class BusinessRequirement(models.Model):
+    _inherit = 'business.requirement'
+
+    sale_order_ids = fields.One2many(
+        comodel_name='sale.order',
+        inverse_name='business_requirement_id',
+        string='Orders',
+    )
+    order_count = fields.Integer(
+        string='Count',
+        compute='_compute_order_count'
+    )
+    detailed_order = fields.Boolean(
+        help='Check this if need generated a detailed order'
+    )
+    product_id = fields.Many2one(
+        comodel_name='product.product',
+        string='Product',
+        domain=[('sale_ok', '=', True)],
+    )
+    qty = fields.Float(
+        string='Quantity',
+        default=1,
+    )
+    sale_price_unit = fields.Float(
+        string='Sales Price',
+    )
+
+    @api.multi
+    @api.onchange('product_id')
+    def product_id_change(self):
+        """Get the right pricelist for product"""
+        uom_id = False
+        sale_price_unit = 0
+        product = self.product_id
+
+        if product:
+            uom_id = product.uom_id.id
+            sale_price_unit = product.list_price
```

Comment:
Here not is 'price'?

Reply:
This is the same criteria as the product on `business.requirement.deliverable` model which for historic reason the default value decided is the product `list_price` if not exist a pricelist. But as the product `price` is computed by Odoo when is passed on the `with_context` function a pricelist so is suitable to be the sale_price_unit of the requirement